src/pyscipopt/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '3.3.0'
+__version__ = '3.5.0'
```

Comment:
This should be 4.0.0 since we are doing a major change to the core functionality of the API, will be changed before merging scip8 to master

Reply:
Now, I have changed it to 4.0.0 👍🏼

Reply:
Now, pyscipopt successfully builds on every os. But the solveConcurrent test fails on macOS with a seg fault. This test only fails on macOS because the linux and windows packages don't install the task processing interface.

Reply:
good to know, for the moment i have disabled the mac test and we can maybe fix it in the near future to reenable it.

Reply:
could you only disable py.test on MacOS (name "Run pyscipopt tests")? the rest works fine on MacOS

Reply:
sure good point